demos/4_Deep_Dives/01_callbacks.ipynb

```diff
@@ -447,13 +446,13 @@
     "            return \" \".join(\"{:>20.5e}\".format(metric) for metric in self.computed_metrics[-len(self.metrics_dict):])\n",
     "        else:\n",
     "            # Handle single metric\n",
-    "            return \"{:>20.5e}\".format(self.computed_metrics)     \n",
-    "        \n",
+    "            return \"{:>20.5e}\".format(self.computed_metrics)\n",
+    "\n",
     "\n",
-    "from cil.utilities.quality_measures import mae, psnr, mse \n",
+    "from cil.utilities.quality_measures import mae, psnr, mse\n",
     "metric_callback= MetricsDiagnostics(ground_truth, {'MSE':mse, 'MAE':mae, 'PSNR':psnr})\n",
-    "algo=FISTA(initial=ig.allocate(0), f=F, g=G, update_objective_interval=10)         \n",
-    "algo.run(100, callbacks=[metric_callback])"
+    "algo=FISTA(initial=ig.allocate(0), f=F, g=G, update_objective_interval=10)\n",
+    "algo.run(100, callbacks=[metric_callback, callbacks.ProgressCallback()])"
```

Comment:
```suggestion
    "algo.run(100, callbacks=[callbacks.ProgressCallback(), metric_callback])"
```
Swapping the order makes the progress bar appear above all of the prints. otherwise it appears after the first line of print-outs